spec2vec/Spec2Vec.py

```diff
@@ -3,14 +3,44 @@
 
 
 class Spec2Vec:
+    """Calculate spec2vec similarity scores between a reference and a query.
 
+    Using a trained model, spectrum documents will be converted into spectrum
+    vectors. The spec2vec similarity is then the cosine similarity score between
+    two spectrum vectors.
+    """
     def __init__(self, model, intensity_weighting_power=0):
+        """
+
+        Parameters
+        ----------
+        model : gensim word2vec model
+            Expecgted input is a gensim word2vec model that has been trained on
+            the desired set of spectrum documents.
+        intensity_weighting_power : float, optional
+            Spectrum vectors are a weighted sum of the word vectors. The given
+            word intensities will be raised to the given power.
+            The default is 0, which means that no weighing will be done.
+        """
         self.model = model
         self.intensity_weighting_power = intensity_weighting_power
         self.vector_size = model.wv.vector_size
 
-    def __call__(self, reference, query):
+    def __call__(self, reference, query) -> float:
```

Comment:
You are using both typings and docstring types, choose one. For example

```suggestion
    def __call__(self, reference: SpectrumDocument, query: SpectrumDocument) -> float:
```

Reply:
I was indeed still not sure what to chose... In principle after working on matchms I got to like the typings, so I now changed it more consistently.
But how do we best handle inputs like a gensim word2vec model? Do I really need to import gensim to specify that type?

Reply:
You need to import, that is how typings work. For a gensim model you could use `from gensim.models.basemodel import BaseTopicModel` to cover more model types.

Reply:
Great, thanks!